README.md

```diff
@@ -7,7 +7,7 @@ This is a demo app of a [unirep](https://github.com/Unirep/Unirep) attester. In
 ## 1. Installation
 
 ```shell
-yarn
+npx create-unirep-app
 ```
```

Comment:
should ask user go to the app directory they create? 😅

Reply:
ok added, we can also have `create-unirep-app` do this automatically, i just think it's a bit jarring if a cli tool changes the working directory on behalf of the user